src/Documentation/UserManual/DeviceOpticalMarkerTracker.dox

```diff
@@ -0,0 +1,86 @@
+/*!
+\page DeviceOpticalMarkerTracker Optical Marker Tracker
+
+\section DeviceOpticalMarkerTrackerSupportedHwDevices Supported hardware devices
+
+- Any external webcam or laptop camera (forward or rear facing).
+
+\section DeviceOpticalMarkerTrackerSupportedPlatforms Included in packages
+
+- \ref PackageWin32
+- \ref PackageWin64
+- \ref PackageInterson
+- \ref PackageBk
+- \ref PackageStealthLink
+- \ref PackageThorLabs
+
+\section DeviceOpticalMarkerTrackerCalibration Calibration
+
+- Print the <a href="https://www.assembla.com/spaces/plus/documents/cd0kq6t7ir577cdmr6QqzO/download/cd0kq6t7ir577cdmr6QqzO">aruco calibration chessboard</a> and measure the side length of the markers in meters.
+- To generate a calibration file, run aruco_calibration.exe using the command
+  \code aruco_calibration.exe live:<camera-index> <output-file-name>.yml -size <marker-size-m> \endcode
+  where:
+  - <camera-index> is the integer index of the camera you want to use (usually 0 or 1).
+  - <output-file-name> is the desired output file name.
+  - <marker-size-mm> is the size of a marker side in meters.
```

Comment:
Is it in mm or m? You used mm in <marker-size-mm> but meters in the description and in the command-line example above is <marker-size-m>